hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -203,19 +204,22 @@ public static Long getMapIdByJobId(String jobId) {
         return createQuery()
                 .select(jobStatus.resourceId)
                 .from(jobStatus)
-                .where(jobStatus.jobId.eq(jobId)).fetchOne();
+                .where(jobStatus.jobId.eq(jobId))
+                .fetchOne();
     }
 
     /**
-     * Gets the job type for the specified jobId
-     * @param jobId
-     * @return
+     * Gets the job id using the map id
+     *
+     * @param mapId map id which was created from a job
+     * @return job id
      */
-    public static Integer getJobTypeByJobId(String jobId) {
```

Comment:
was this also unused?

Reply:
yup. been trying to do some cleanup along the way if i make changes to a specific file